doc/tutorials/ann/ann.txt

```diff
@@ -79,9 +79,8 @@ have a number of methods in common:
 
 @note
 To be able to optimize the network, both classes implement the OptimizerFunction
-API; see \ref optimizertutorial "Optimizer API" for more information. In short,
```

Comment:
What tutorial is this meant to be?

Reply:
That is an artifact from the time when ensmallen was part of the mlpack/mlpack repo. Save to delete the reference.